py/defs.bzl

```diff
@@ -73,14 +73,14 @@ def py_binary(name, srcs = [], main = None, imports = ["."], **kwargs):
 
     _py_binary_or_test(name = name, rule = _py_binary, srcs = srcs, main = main, imports = imports, resolutions = resolutions, **kwargs)
 
-def py_test(name, main = None, srcs = [], imports = ["."], **kwargs):
+def py_test(name, main = None, srcs = [], imports = [], **kwargs):
     "Identical to py_binary, but produces a target that can be used with `bazel test`."
 
     # Ensure that any other targets we write will be testonly like the py_test target
     kwargs["testonly"] = True
     _py_binary_or_test(name = name, rule = _py_test, srcs = srcs, main = main, imports = imports, **kwargs)
 
-def py_library(name, imports = ["."], **kwargs):
+def py_library(name, imports = [], **kwargs):
```

Comment:
Given that nothing reads `imports` now, let's remove it from the method signatures and default it to an empty list on the rule attr. It can be passed here via kwargs.